test/e2e/storage/csi_mock_volume.go

```diff
@@ -1301,8 +1301,7 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 						framework.ExpectNoError(err, "Failed to get claim: %v", err)
 					}
 					framework.Logf("PVC not found. Continuing to test VolumeSnapshotContent finalizer")
-				}
-				if claim != nil && claim.DeletionTimestamp == nil {
+				} else if claim.DeletionTimestamp == nil {
```

Comment:
this can avoid test failure. But I roughly checked the test code, it is kind of strange for me this test creates snapshot first, and then PVC? Could you please double check?

Reply:
Yes, they are creating snapshot first and then PVC. Does that matter? I think it is okay. The snapshot will just waiting I guess?

Reply:
ok, I think the original design is snapshot will fail, but now I kind of remember the design is changed to retry forever..